src/WebApiToTypeScript/WebApi/WebApiAction.cs

```diff
@@ -172,8 +176,8 @@ public ConstructorParameterMapping[] GetConstructorParameterMappings()
                     IsOptional = routePart.IsOptional && TypeService.IsParameterOptional(routePart.Parameter),
                     TypeMapping = routePart.GetTypeMapping(),
                     Name = routePart.Parameter.Name,
-                    StringWithOptionals = routePart.GetParameterString(interfaceName: true),
-                    String = routePart.GetParameterString(withOptionals: false, interfaceName: true)
+                    StringWithOptionals = routePart.GetParameterString(interfaceName: true, ignoreEnumDefinitions: this.IsMobileAction),
```

Comment:
@dsperac In this file, `WebApiAction`, seems like the fact that it is mobile only makes the route parts ignore enums, so can we just call the property `IgnoreEnumDefinitions` instead? It will read nicer and express the intent? What do you think?

Reply:
agreed generally, but if we are going with your point #1 from above (setting the prop once and then filtering by it), filtering by "IgnoreEnumDefinitions" is a bit weird, so I would leave IsMobileAction